Misc/python.man

```diff
@@ -447,83 +476,77 @@ needed for developing Python extensions and embedding the
 interpreter.
 .RE
 .SH ENVIRONMENT VARIABLES
-.IP PYTHONSAFEPATH
-If this is set to a non-empty string, don't automatically prepend a potentially
-unsafe path to \fBsys.path\fP such as the current directory, the script's
-directory or an empty string. See also the \fB\-P\fP option.
-.IP PYTHONHOME
-Change the location of the standard Python libraries.  By default, the
-libraries are searched in ${prefix}/lib/python<version> and
-${exec_prefix}/lib/python<version>, where ${prefix} and ${exec_prefix}
-are installation-dependent directories, both defaulting to
-\fI/usr/local\fP.  When $PYTHONHOME is set to a single directory, its value
-replaces both ${prefix} and ${exec_prefix}.  To specify different values
-for these, set $PYTHONHOME to ${prefix}:${exec_prefix}.
-.IP PYTHONPATH
-Augments the default search path for module files.
-The format is the same as the shell's $PATH: one or more directory
-pathnames separated by colons.
-Non-existent directories are silently ignored.
-The default search path is installation dependent, but generally
-begins with ${prefix}/lib/python<version> (see PYTHONHOME above).
-The default search path is always appended to $PYTHONPATH.
-If a script argument is given, the directory containing the script is
-inserted in the path in front of $PYTHONPATH.
-The search path can be manipulated from within a Python program as the
-variable
-.IR sys.path .
-.IP PYTHONPLATLIBDIR
-Override sys.platlibdir.
-.IP PYTHONSTARTUP
-If this is the name of a readable file, the Python commands in that
-file are executed before the first prompt is displayed in interactive
-mode.
-The file is executed in the same name space where interactive commands
-are executed so that objects defined or imported in it can be used
-without qualification in the interactive session.
-You can also change the prompts
-.I sys.ps1
-and
-.I sys.ps2
-in this file.
-.IP PYTHONOPTIMIZE
-If this is set to a non-empty string it is equivalent to specifying
-the \fB\-O\fP option. If set to an integer, it is equivalent to
-specifying \fB\-O\fP multiple times.
+.IP PYTHONASYNCIODEBUG
+If this environment variable is set to a non-empty string, enable the debug
+mode of the asyncio module.
+.IP PYTHON_BASIC_REPL
+If this variable is set to any value, the interpreter will not attempt to
+load the Python-based REPL that requires curses and readline, and will instead
+use the traditional parser-based REPL.
+.IP PYTHONBREAKPOINT
+If this environment variable is set to 0, it disables the default debugger. It
+can be set to the callable of your debugger of choice.
+.IP PYTHONCOERCELOCALE
```

Comment:
Let's also fix `main`:
```suggestion
.IP PYTHONCOERCECLOCALE
```
(This is one reason why we add underscores for new env vars :)